### PR TITLE
More changes to reshaping

### PR DESCRIPTION
**Pull request by ian-rickard**
I'm sorry for changing this again Chloe. Can you have a look and let me know if it seems ok? I wanted to make it a little more interactive. Also there were a few stray annotations in the code that needed getting rid of.

### CODE REVIEW

File: rmd_files/reshaping.Rmd [hunk 1]
```diff
@@ -169,24 +171,27 @@ head(long_annual_offences)
...
 
 ```{r, purl=purl_example_code}
 head(annual_offences, 3)
 head(long_annual_offences, 3)
 ```
 
-```{r, purl=purl_example_code}
-identical(long_annual_offences, annual_offences)
-```
+In fact, there are *six* differences between these tables. Have a look yourself, and put suggestions in the chat as to what these might be. Then we'll cover how to correct these differences and make our  working table identical to our working table.
```
Comment (chloepugh) [inline rmd_files/reshaping.Rmd:185]: Is there a mistake where it says "make our working table identical to our working table"?

File: rmd_files/reshaping.Rmd [hunk 2]
```diff
@@ -330,15 +339,22 @@ reoffending_real <- Rs3tools::s3_path_to_full_df(
...
+  c) What, if anything, would you need to do to the data before passing it to be read for plotting by functions from a package like `ggplot2`?
 
- a) Put the data in long format
- b) Remove relevant prefixes
- c) Pass the labels 'quarter' and 'count' to the appropriate arguments to name the columns in your long format table.
+Note, these are real data on reoffending, publicly available, derived from the table [here](https://www.gov.uk/government/statistics/proven-reoffending-statistics-april-to-june-2021).
+
+---
+
+### Exercise 2
```
Comment (chloepugh) [inline rmd_files/reshaping.Rmd:350]: Is it worth putting Exercise 2 after the data preview below?

**Review by chloepugh — COMMENTED** [review]
No problem @ian-rickard! I don't think this has affected the slides much, just a couple of minor comments below

File: rmd_files/reshaping.Rmd (hunk 1 above)
Comment (ian-rickard) [reply rmd_files/reshaping.Rmd:185]: good catch, will fix

File: rmd_files/reshaping.Rmd (hunk 2 above)
Comment (ian-rickard) [reply rmd_files/reshaping.Rmd:350]: yes, good idea, will do!

**Pull request merged**